### ISSUE TITLE
Instant crash when selecting particle effect.

### ISSUE THREAD

**Issue by Retrogamebloke** [TheGameCreators/GameGuruRepo#2641]
I've just tried a demo I made a couple of months back. All runs fine when I run it, but if I try to click on the particle effect in the scene it crashes out completely. I get it every time.
![image](https://user-images.githubusercontent.com/44620846/159977908-337a8a0c-a4d1-4aac-ab98-13a2b9fbab77.png)
I have sent the map to Lee ages ago, for other reasons, but can send again if needed. I haven't tried deleting the effect and re-adding it as yet, but thought this seems worth investigating if possible before release if not just down to an old map.

**Comment by dllllllllllb** [comment]
@Retrogamebloke Please send your level to me so I can test it for Lee, my email is: maciej@thegamecreators.com

**Comment by LeeBamberTGC** [comment]
@dllllllllllb Once you get the level and reproduce, send it to me and I will see about a fix :)

**Comment by Retrogamebloke** [comment]
I sent the level yesterday, let me know if you need anything else.

**Comment by dllllllllllb** [comment]
@Retrogamebloke Thanks for sending me your level. I'm not getting a crash after selecting the particle.

Is this issue only occurring in this level for you?

**Comment by Retrogamebloke** [comment]
Yes. Not seen it on other levels, but have tried this on the release version with the same result with this  map.

**Comment by PaulSJ** [comment]
@Retrogamebloke Is this still an issue with the latest version?

**Comment by LeeBamberTGC** [comment]
@Retrogamebloke I will close this, and if you still see this in an older imported level, please email me a fresh copy of it and so I can find the cause, thanks!
